connectors/ssh/src/main/java/org/crsh/auth/FilePublicKeyProvider.java

```diff
@@ -59,7 +60,7 @@
     List<KeyPair> keys = new ArrayList<KeyPair>();
     for (String file : files) {
       try {
-          Object o = KeyPairUtils.readKey(new InputStreamReader(new FileInputStream(file)));
+          Object o = KeyPairUtils.readKey(new InputStreamReader(new FileInputStream(file), Charset.forName("UTF-8")));
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.